Assets/Mirror/Editor/Weaver/Processors/NetworkBehaviourProcessor.cs

```diff
@@ -107,10 +107,17 @@ public static void WriteSetupLocals(ILProcessor worker)
         public static void WriteCreateWriter(ILProcessor worker)
         {
             // create writer
-            worker.Append(worker.Create(OpCodes.Newobj, Weaver.NetworkWriterCtor));
+            worker.Append(worker.Create(OpCodes.Call, Weaver.GetPooledWriterReference));
```

Comment:
@vis2k  this is the call to GetWriter().   This class has 2 methods:  

* `WriteCreateWriter` which was already being called everywhere, and now calls `NeworkWriterPool.GeWriter()`
* `WriteRecyleWriter` is being introduced in this PR,  and it is now called everywhere where WriteCreateWriter was already being called.

Reply:
that makes perfect sense